feature_list/feature_list.cpp

```diff
@@ -83,6 +84,25 @@ string GetReadableType(FeatureType const & f)
   return result == 0 ? string() : classif().GetReadableObjectName(result);
 }
 
+string GetWheelchairType(FeatureType const & f)
```

Comment:
Ah, see now. As this is not a library, okay.

Meanwhile, it's a good practice to put all what you don't want to expose outside to the anonymous namespaces, even in main files.